usefulUtilities/sitV2/appResources/appData/sitClientV2.js

```diff
@@ -0,0 +1,769 @@
+            }
+
+            // RESTORE ANIMATION ROLES
+            var roles = rolesToOverride();
+            for (var j in roles) {
+                MyAvatar.restoreRoleAnimation(roles[j]);
+            }
+
+            // If changed seat do not do this
+            MyAvatar.collisionsEnabled = true;
+            MyAvatar.hmdLeanRecenterEnabled = true;
+            Script.setTimeout(function () {
+                MyAvatar.centerBody();
+            }, STANDUP_DELAY_MS);
+
+            // SET AVATAR POSITION AND ORIENTATION TO PREVIOUS VALUES
+            var avatarDistance = Vec3.distance(MyAvatar.position, _this.seatCenterPosition);
+            var properties = Entities.getEntityProperties(_this.entityID);
+            if (avatarDistance < STANDUP_DISTANCE_M) { // might need to 
+                // Avatar did not teleport far away from chair
+                // Therefore apply previous position OR the default dismount position and orientation
+                if (settingsPreviousPosition && settingsPreviousOrientation) {
+                    // Apply previous position and orientation because they exist
+                    MyAvatar.position = settingsPreviousPosition;
+                    MyAvatar.orientation = settingsPreviousOrientation;
+                } else {
+                    // Calculate chair default dismount position in front of chair
+                    var offset = {
+                        x: 0,
+                        y: 1.0,
+                        z: CHAIR_DISMOUNT_OFFSET_M - properties.dimensions.z * properties.registrationPoint.z
+                    };
+                    var position = Vec3.sum(properties.position, Vec3.multiplyQbyV(properties.rotation, offset));
+                    MyAvatar.position = position;
+                }
+            }
+            Settings.setValue(SETTING_KEY_AVATAR_SITTING, "");
+        }
+
+        // RESET SETTINGS FOR THIS CHAIR
+        // Could have changed seats, keep avatar sitting if did not go through above procedure
+        if (!_this.locked) {
+            Entities.editEntity(_this.entityID, { locked: false });
+        }
+
+        _this.driveKeyPressedStart = false;
+        _this.sitDownSettlePeriod = false;
+        _this.isSittingDown = false;
+        deleteStandUp();
+
+        Entities.callEntityServerMethod(_this.entityID, "onStandUp");
+
+        stopUpdateInterval();
+
+        if (_this.connectedSignals) {
+            MyAvatar.scaleChanged.disconnect(_this.standUp);
+            MyAvatar.onLoadComplete.disconnect(_this.standUp);
+            location.hostChanged.disconnect(_this.standUp);
+            Script.scriptEnding.disconnect(_this.standUp);
+            MyAvatar.skeletonModelURLChanged.disconnect(_this.standUp);
+            MyAvatar.wentAway.disconnect(_this.standUp);
+            _this.connectedSignals = false;
+        }
+
+        // RESET OVERLAYS FOR ALL AVATARS IN RANGE OF THE CHAIR
+        Script.setTimeout(function () {
```

Comment:
why does there need to be a timeout here?

Reply:
investigating..

Reply:
It waits for the sitting avatar to dismount away from the chair before rendering the sittable overlay (one conditional is it will NOT render if an avatar is <0.01 away from the chair center). 

During my testing, it was not reliable if there was a timeout less than 200 ms. 500 worked every time.